cmd/oracled/cmd/register_oracle.go

```diff
@@ -112,7 +112,9 @@ func sendTxRegisterOracle(cmd *cobra.Command, conf *config.Config, svc service.S
 	return nil
 }
 
-func generateMsgRegisterOracle(cmd *cobra.Command, conf *config.Config, oracleAccount *panacea.OracleAccount, trustedBlockInfo *panacea.TrustedBlockInfo) (*oracletypes.MsgRegisterOracle, error) {
+func generateMsgRegisterOracle(cmd *cobra.Command, svc service.Service, oracleAccount *panacea.OracleAccount, trustedBlockInfo *panacea.TrustedBlockInfo) (*oracletypes.MsgRegisterOracle, error) {
```

Comment:
Inside the service, it contains both config and sgx, so change it to receive this service.